src/DacpacTool/PackageBuilder.cs

```diff
@@ -50,17 +50,27 @@ private void ValidateReference(string referenceFile)
             }
         }
 
-        public void AddSqlCmdVariables(string[] variableNames)
+        public void AddSqlCmdVariables(string[] variables)
         {
             // Ensure that the model has been created
             EnsureModelCreated();
 
-            foreach (var variableName in variableNames)
+            foreach (var variable in variables)
```

Comment:
Seems like this `foreach` loop only exists for logging purposes now, so we're essentially enumerating the list of variables twice. Perhaps we should move this logging into the `AddSqlCmdVariables()` method in `Extensions` instead?

Reply:
What about keeping this loop and removing the one in `Extensions`, with a new `AddSqlCmdVariable(string Name, string DefaultValue)` method to 'Extensions'? Feels cleaner to me.

Reply:
Yeah, I agree that does feel cleaner.